kernel/calls.c

```diff
@@ -6,10 +6,13 @@
 #include "kernel/signal.h"
 #include "kernel/task.h"
 
-dword_t syscall_stub() {
+dword_t syscall_stub(void) {
     return _ENOSYS;
 }
-dword_t syscall_success_stub() {
+dword_t syscall_silent_stub(void) {
```

Comment:
Should have a comment explaining how one of these is silent and one isn't. Or just move the stub syscall printk into the actual syscall stub function tbh